imap_processing/swe/l1b/swe_l1b_science.py

```diff
@@ -359,6 +362,9 @@ def swe_l1b_science(l1a_data):
     l1a_data_copy = l1a_data.copy(deep=True)
 
     full_cycle_data_indices = get_indices_of_full_cycles(l1a_data["QUARTER_CYCLE"].data)
+    logger.info(
```

Comment:
This is more of a debug-level log I think. I'm not sure we want to be logging the data every time during processing. Or just remove them if it was just for quick testing.
```suggestion
    logger.debug(
```